backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanIrModuleSerializer.kt

```diff
@@ -1,49 +1,40 @@
 package org.jetbrains.kotlin.backend.konan.serialization
 
 import org.jetbrains.kotlin.backend.common.LoggingContext
-import org.jetbrains.kotlin.backend.common.descriptors.propertyIfAccessor
 import org.jetbrains.kotlin.backend.common.serialization.*
-import org.jetbrains.kotlin.backend.konan.descriptors.isFromInteropLibrary
+import org.jetbrains.kotlin.backend.common.serialization.signature.IdSignatureSerializer
 import org.jetbrains.kotlin.descriptors.DeclarationDescriptor
-import org.jetbrains.kotlin.ir.declarations.IrDeclaration
 import org.jetbrains.kotlin.ir.declarations.IrFile
 import org.jetbrains.kotlin.ir.descriptors.IrBuiltIns
 import org.jetbrains.kotlin.ir.symbols.IrSymbol
-import org.jetbrains.kotlin.ir.util.UniqId
-import org.jetbrains.kotlin.resolve.descriptorUtil.module
 
-private class KonanDeclarationTable(
-        descriptorTable: DescriptorTable,
-        globalDeclarationTable: GlobalDeclarationTable,
-        startIndex: Long
-) : DeclarationTable(descriptorTable, globalDeclarationTable, startIndex),
-        DescriptorUniqIdAware by DeserializedDescriptorUniqIdAware {
+private class KonanDeclarationTable(globalDeclarationTable: GlobalDeclarationTable) :
+        DeclarationTable(globalDeclarationTable) {
 
     /**
      * It is incorrect to compute UniqId for declarations from metadata-based libraries.
      * Instead we should get precomputed value from metadata.
      */
-    override fun tryComputeBackendSpecificUniqId(declaration: IrDeclaration): UniqId? {
-        return if (declaration.descriptor.module.isFromInteropLibrary()) {
-            // Property accessor doesn't provide UniqId so we need to get it from the property itself.
-            UniqId(declaration.descriptor.propertyIfAccessor.getUniqId() ?: error("No uniq id found for ${declaration.descriptor}"))
-        } else {
-            null
-        }
-    }
+//    override fun tryComputeBackendSpecificUniqId(declaration: IrDeclaration): UniqId? {
```

Comment:
We usually do not leave commented out code. Why it cannot be just removed?

Reply:
I've thought mirror contains reviewed commits :/